src/EventExport/Format/HTML/HTMLEventFormatter.php

```diff
@@ -144,13 +150,52 @@ public function formatEvent($eventString)
 
     /**
      * @param stdClass $event
+     * @return string
+     */
+    private function getEventId(stdClass $event)
+    {
+        $urlParts = explode('/', $event->{'@id'});
+        return end($urlParts);
+    }
+
+    /**
+     * @param stdClass $event
+     * @param array $formattedEvent
+     */
+    private function addCalendarInfo(stdClass $event, array &$formattedEvent)
+    {
+        // Set the pre-formatted calendar summary as fallback in case no calendar repository was provided.
+        $formattedEvent['dates'] = $event->calendarSummary;
+
+        $calendar = null;
+        $formatter = null;
+
+        if ($this->calendarRepository) {
+            $eventId = $this->getEventId($event);
+            $calendar = $this->calendarRepository->get($eventId);
+        }
+
+        if ($calendar instanceof \CultureFeed_Cdb_Data_Calendar_TimestampList) {
```

Comment:
Can we use the CalendarHTMLFormatter class instead which handles this logic?
https://github.com/cultuurnet/calendar-summary/blob/master/src/CalendarHTMLFormatter.php